python/ray/autoscaler/tags.py

```diff
@@ -1,22 +1,23 @@
+"""The Ray autoscaler uses tags/labels to associate metadata with instances."""
+
 from __future__ import absolute_import
 from __future__ import division
 from __future__ import print_function
-"""The Ray autoscaler uses tags to associate metadata with instances."""
 
 # Tag for the name of the node
-TAG_NAME = "Name"
-
-# Tag uniquely identifying all nodes of a cluster
-TAG_RAY_CLUSTER_NAME = "ray:ClusterName"
+TAG_RAY_NODE_NAME = "ray-node_name"
```

Comment:
How about we go with just underscores, e.g. `ray_node_name`. This would be in case dashes become a problem in the future, and also to avoid mixing both dashes and underscores.

Reply:
Good idea. Changed in next commit.

Reply:
I just realized that the label keys can contain underscores but the label values cannot. The values must be a match of regex: `(?:[a-z](?:[-a-z0-9]{0,61}[a-z0-9])?)`. Let's use dashes instead.